### DISCUSSION TITLE
Toggle for automatically closing dropdown menu when in 'enlarged UI'

### DISCUSSION THREAD

**Discussion by SixFrags** [OpenRCT2/OpenRCT2#22137] [Feature requests] [answered]
I really love the enlarged UI option, but the dropdown menu staying open after clicking for example a landtype or rotating the view is an annoyance for me. 
Would be great if you can turn off the dropdown menu staying open in enlarged UI.

To clarify, when building and want to quickly rotate the view with a mouseclick, now the dropdown menu stays open and you have to click the 'Rotate clockwise' option: 
![12](https://github.com/OpenRCT2/OpenRCT2/assets/83868085/00e9e0fa-4777-4a9f-930d-80d99265807a)

Another example is that when you want to quickly repaint a landtile with the normal grass color with 1 mouseclick, you'll now get the dropdown menu and have to select the normal grass again. In long term this can become an annoyance:
![14](https://github.com/OpenRCT2/OpenRCT2/assets/83868085/a9a43eb0-d768-4156-b6a9-ab3e58cbd213)

**Comment by FischNE** [comment]
Great suggestion, this would reaaaally save clicks!!

**Comment by Gymnasiast** [comment, marked as answer]
This has now been implemented.
